homeassistant/components/combined_energy/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "combined_energy",
+  "name": "Combined Energy",
+  "codeowners": ["@timsavage"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/combined_energy",
+  "integration_type": "hub",
+  "iot_class": "cloud_polling",
+  "quality_scale": "silver",
```

Comment:
Please remove the quality scale for now. You can add it in a follow up PR later on to keep this PR as small as possible
```suggestion
```